index.html

```diff
@@ -12,29 +12,31 @@
     <div id="appFrame">
       <section id="pageNavContainer"></section>
       <section id="contentFrame">
-        <section id="pageContainer"></section>
-        <section id="chatContainer">
-          <div class="padSmKids">
-            <div>
-              <a class="discSm border clrBr js-navListBtn" style="background-image: url('../imgs/defaultUser.png')"></a>
-            </div>
-            <div>
-              <a class="discSm border clrBr js-navListBtn" style="background-image: url('../imgs/defaultUser.png')"></a>
-            </div>
-            <div>
-              <a class="discSm border clrBr js-navListBtn" style="background-image: url('../imgs/defaultUser.png')"></a>
-            </div>
-            <div>
-              <a class="discSm border clrBr js-navListBtn" style="background-image: url('../imgs/defaultUser.png')"></a>
-            </div>
-            <div>
-              <a class="discSm border clrBr js-navListBtn" style="background-image: url('../imgs/defaultUser.png')"></a>
-            </div>
+        <div id="chatLeftSpacer"></div>
+        <div id="pageContainer"></div>
+        <div id="chatRightSpacer"></div>
```

Comment:
The spacers allow some flex magic to work for the positioning of the pageContainer. The left spacer will collapse if the window is too small, allowing the pageContainer element to move to the left. 

If the window isn't too small, the pageContainer will align with the left edge of the tabs.